lib/msal-core/samples/quickStart/index_LoginRedirect.html

```diff
@@ -0,0 +1,86 @@
+<html>
+<head>
+    <title>Quickstart for Microsoft authentication libray for JS</title>
+</head>
+<body>
+<!-- bluebird only needed if this page needs to run on Internet Explorer -->
+<script src="https://cdnjs.cloudflare.com/ajax/libs/bluebird/3.3.4/bluebird.min.js" class="pre"></script>
+<script src="https://secure.aadcdn.microsoftonline-p.com/lib/0.2.1/js/msal.min.js"></script>
+<script src="https://code.jquery.com/jquery-3.2.1.min.js" class="pre"></script>
+
+<div>
+    <div id="label">Sign-in with Microsoft</div>
+    <br>
+    <button id="auth" onclick="loginRedirect();">Login (with Redirect)</button>
+</div>
+
+<pre class="response"></pre>
+
+<script class="pre">
```

Comment:
Is it possible to separate out the JS code into an app.js separate from the index.html?

Reply:
I think the idea was to make it a single file and as less code as possible.  There isn't much code in the html to separate it. WE can discuss this offline.